src/Core/Aetherling/Passes/ThroughputModifications.hs

```diff
@@ -2,69 +2,57 @@
 Module: Aetherling.Passes.ThroughputModifications
 Description: Passes that tradeoff throughput and area
 
-The two main passes that modify the throughput of Aetherling operations are
```

Comment:
The comments in dev are the latest version. The version of this file in NewOps is an older version of this file. Make sure you are working on the latest version of this file.